components/monitoring/network/network.js

```diff
@@ -117,13 +127,37 @@ function activateNetwork() {
 
 function refreshNetwork() {
     updateNetworkUsage();
+    checkConnectivity();
+}
+
+function checkConnectivity(){
+    si.inetChecksite('https://google.com')
```

Comment:
I don't approve of connecting to google just to check if the user has internet.
Surely there must be a way on each platform to check if the network interface has internet access.

Reply:
Especially combined with the fact that it connects on every interval :slightly_frowning_face: